geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -192,9 +194,20 @@ private void loadWebApplications() {
         logger.debug(message);
       }
     } else {
+      String[] authTokenEnabledComponents = config.getSecurityAuthTokenEnabledComponents();
+      boolean pulseOauth = Arrays.stream(authTokenEnabledComponents)
```

Comment:
Since you only use "pulseOauth" inside the if(isIntegratedSecurity) would it be better to initialize the variable inside that if?